app/services/immunization_service/reports/general/vaccines_administered.rb

```diff
@@ -37,15 +37,18 @@ def calculate_age(birthdate, reference_date)
         end
         
         def generate(start_date, end_date)
+          batch_number_concept_id = ConceptName.find_by_name('Batch Number').concept_id
+
           base_query = Order.joins(:encounter)
```

Comment:
This needs to be pulling from obs so that the tally with numbers that are administered